tdrs-backend/tdpservice/core/models.py

```diff
@@ -0,0 +1,40 @@
+"""Core models."""
+
+from django.contrib.auth.models import Permission
+from django.contrib.contenttypes.models import ContentType
+from django.db import models
+
+"""Global permissions
```

Comment:
@carltonsmith We discussed this last week — it's not clear to me if global permissions are being used at all in the application right now. 

If global permissions are being used, could you point me to where in the app they're being used?

If not, I think we should back this out and remove the files in this PR that introduced global permissions.

Reply:
@alexsoble While we did talk about creating endpoints for permissions, I don't recall discussing this directly. This is not to support permission endpoints, but to support any permissions that are not directly related to a specific model. Django, by default, has permissions for things such as "Can edit User" and "Can create STT", but for any permissions that are not directly related to a model, such as being able to view partial pages, not seeing particular links and such, global permissions are required. This is the base implementation to allow for that down the road.

Perhaps it is just a question of the approach one takes to programming, but even in an agile environment, I prefer to include the basic parts of something I know will be built upon and expanded in the future, because I know the basic architecture is there. I have run into issues in the past where leaving out the basics causes problems down the road. In my view, this is one of those circumstances and I would really prefer to not leave it out at this point.

Reply:
Agreed that this is probably a difference in agile philosophy! The QASP doesn't speak to this directly one way or the other, since you feel that this will help future devs I'm happy to keep it in. 😄

Reply:
My only thought is, might be worth adding a code comment explaining that this model isn't used yet for future developers trying to understand the permission structure

Reply:
This is all set

Reply:
Great!